cluster-autoscaler/core/autoscaler.go

```diff
@@ -127,7 +127,7 @@ func initializeDefaultOptions(opts *AutoscalerOptions, informerFactory informers
 		opts.FrameworkHandle = fwHandle
 	}
 	if opts.ClusterSnapshot == nil {
-		opts.ClusterSnapshot = predicate.NewPredicateSnapshot(base.NewBasicClusterSnapshot(), opts.FrameworkHandle)
+		opts.ClusterSnapshot = predicate.NewPredicateSnapshot(base.NewBasicSnapshotBase(), opts.FrameworkHandle)
```

Comment:
Same comment as to others - I think `NewBasicClusterStorage` would be better, or `NewBasicClusterState`. `SnapshotBase` does not really mean anything.